source/assets/js/components/fef-generic-expander.js

```diff
@@ -0,0 +1,138 @@
+import {KEYCODES} from '../utils/fef-keycodes';
+
+const HOOK_CLASS        = '.js-genex';
+const TOGGLE_CLASS      = '.js-genex-toggle';
+const ANIMATION_DEFAULT_EASING = 'easeInOutCubic';
+
+let ANIMATION_DEFAULT_DURATION = 400;
+
+if (window.matchMedia('(prefers-reduced-motion)').matches) {
+    ANIMATION_DEFAULT_DURATION = 0;
+}
+
+export function init() {
+    $(HOOK_CLASS).each((index, element) => {
+        new FefGenericExpander(element);
+    });
+}
+export class FefGenericExpander {
+    constructor(element) {
+        this.$element = $(element);
+        this.openToggleClass = this.$element.attr('data-genex-open-toggle-class') || 'js-genex-toggle-open';
+        this.openPanelClass  = this.$element.attr('data-genex-open-panel-class')  || 'js-genex-panel-open';
+
+        this.eventSource = this.$element.attr('data-event-source');
+        this.eventValue = this.$element.attr('data-event-value');
+
+        this.initA11Y();
+        this.bindEvents();
+
+        this.isTogglingAllowed = true;
+    }
+
+    initA11Y() {
+        // set aria-expand attribute
+        $(TOGGLE_CLASS).attr('aria-expanded', false);
+
+        // set aria-controls attribute
+        $('[data-genex-target-id]').each((index, element) => {
+            $(element).attr('aria-controls', $(element).attr('data-genex-target-id'));
+        });
+    }
+
+    bindEvents() {
+        let clickHandler = (event) => {
+            event.preventDefault();
+            this.togglePanels(event);
+        };
+
+        let keyboardHandler = (event) => {
+            if (event.keyCode === KEYCODES.enter || event.keyCode === KEYCODES.space) {
+                event.preventDefault();
+                this.togglePanels(event);
+            }
+        };
+
+        this.$element.on('click', TOGGLE_CLASS, clickHandler);
+        this.$element.on('keydown', TOGGLE_CLASS, keyboardHandler);
+    }
+
+    togglePanels(event) {
+        //prevent toggling of multiple panels
+        if (this.isTogglingAllowed) {
+            let isSwitching = false;
+
+            const $lastToggle = $('.' + this.openToggleClass, this.$element);
+            const $currentToggle = $(event.currentTarget);
+            const $openPanel = $('.' + this.openPanelClass, this.$element);
```

Comment:
```suggestion
            const $openPanel = $(`.${this.openPanelClass}`, this.$element);
```
(persönliche Preferenz)